zuul-core/src/test/java/com/netflix/zuul/netty/common/proxyprotocol/ElbProxyProtocolChannelHandlerTest.java

```diff
@@ -76,11 +100,11 @@ public void negotiateProxy_ppv1_ipv4() {
 
         // The handler should remove itself.
         assertNull(channel.pipeline().context(ElbProxyProtocolChannelHandler.NAME));
-        assertEquals(
-                HAProxyProtocolVersion.V1, channel.attr(ElbProxyProtocolChannelHandler.ATTR_HAPROXY_VERSION).get());
+        assertNull(channel.pipeline().context("HAProxyMessageChannelHandler"));
```

Comment:
This should be the class passed in, not the string.

Reply:
Thanks. Much cleaner API that. Done.